package.json

```diff
@@ -48,8 +47,8 @@
       "TestDelayOrError": "components/TestDelayOrError.coffee"
     },
     "graphs": {
-      "RepeatTest": "spec/fixtures/repeat.fbp",
-      "IIPKickTest": "spec/fixtures/kickiip.fbp"
+      "RepeatTest": "graphs/repeat.fbp",
```

Comment:
Why is this? Fine to have a convention, but to have hardcoded expectations is not cool - especially when it used to not be.